src/modules/Profile.rb

```diff
@@ -13,26 +13,10 @@ class ProfileClass < Module
     # All these sections are handled by AutoYaST (or Installer) itself,
     # it doesn't use any external AutoYaST client for them
     GENERIC_PROFILE_SECTIONS = [
-      # Booting is configured during the installation
-      "bootloader",
```

Comment:
A good point. Well then I am tending to let bootloader, add-on, kdump, deploy_image in
GENERIC_PROFILE_SECTIONS.